media_softlet/linux/Xe_M_plus/ddi/media_libva_caps_mtl_base.cpp

```diff
@@ -99,11 +99,11 @@ const VAImageFormat m_supportedImageformatsXe_Lpm_Plus_Base[] =
     (fourcc_mod_code(INTEL, (val)) | INTEL_PRELIM_ID_FLAG)
 
 /* this definition is to avoid duplicate drm_fourcc.h this file is updated seldom */
-#ifndef PRELIM_I915_FORMAT_MOD_4_TILED_MTL_MC_CCS
-#define PRELIM_I915_FORMAT_MOD_4_TILED_MTL_MC_CCS    intel_prelim_fourcc_mod_code(17)
+#ifndef I915_FORMAT_MOD_4_TILED_MTL_MC_CCS
```

Comment:
These self-definitions should be completely pruned ASAP. Definitions from i915 header files driver carries should be used instead.